src/js/components/Tabs/Tabs.js

```diff
@@ -114,7 +83,21 @@ const Tabs = forwardRef(
           gap={theme.tabs.gap}
           {...tabsHeaderStyles}
         >
-          {tabs}
+          {React.Children.map(children, (child, index) => (
+            <TabsContext.Provider
+              // eslint-disable-next-line react/no-array-index-key
+              key={index}
```

Comment:
`key` does not need to be specified here. `React.Children.map()` is already generating unique keys. So `key` can be removed.
```suggestion
```

Reply:
Good clean up, thanks!